pdfa/pdf_apply_standard.go

```diff
@@ -0,0 +1,77 @@
+/*
+ * PDF optimization (compression) example.
+ *
+ * Run as: go run pdf_apply_standard.go <input.pdf> <output.pdf>
+ */
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/unidoc/unipdf/v3/common/license"
+	"github.com/unidoc/unipdf/v3/model"
+	"github.com/unidoc/unipdf/v3/model/pdfa"
+)
+
+const licenseKey = `
+-----BEGIN UNIDOC LICENSE KEY-----
+Free trial license keys are available at: https://unidoc.io/
+-----END UNIDOC LICENSE KEY-----
+`
+
+const usage = "Usage: %s INPUT_PDF_PATH OUTPUT_PDF_PATH\n"
+
+func init() {
+	err := license.SetLicenseKey(licenseKey, `Company Name`)
+	if err != nil {
+		panic(err)
+	}
+}
+
+func main() {
+	args := os.Args
+	if len(args) < 3 {
+		fmt.Printf(usage, os.Args[0])
+		return
+	}
+	inputPath := args[1]
+	outputPath := args[2]
+
+	// Initialize starting time.
+	start := time.Now()
+
+	// Create reader.
+	inputFile, err := os.Open(inputPath)
+	if err != nil {
+		log.Fatalf("Fail: %v\n", err)
+	}
+	defer inputFile.Close()
+
+	reader, err := model.NewPdfReader(inputFile)
```

Comment:
can use model.NewPdfReaderFromFile(inputFile)... slightly shorter :).
So we don't need to open the reader in compliance mode to apply?  ok cool

Reply:
Sure :+1: 
Done. Yeah, it is not required by the writer. It mostly is used by the PdfReader to extract some metadata from the document.